activesupport/test/cache/stores/mem_cache_store_test.rb

```diff
@@ -197,4 +224,24 @@ def emulating_unavailability
       Dalli.send(:remove_const, :Server)
       Dalli.const_set(:Server, old_server)
     end
+
+    def servers(cache = @cache)
+      client(cache).instance_variable_get(:@servers)
+    end
+
+    def client(cache = @cache)
+      cache.instance_variable_get(:@data)
+    end
```

Comment:
I didn't feel great about doing this, but neither of these are publicly accessible, and we were already using `cache.instance_variable_get(:@data)` in a couple tests.